tapdb/sqlc/queries/universe.sql

```diff
@@ -116,15 +116,15 @@ WITH group_key_root_id AS (
     LIMIT 1
 )
 INSERT INTO universe_events (
-    event_type, universe_root_id, event_time
+    event_type, universe_root_id, event_time, event_timestamp
```

Comment:
What's the difference between `event_time` and `event_timestamp`?

Reply:
Just the format. One is the Golang string representation of a `time.Time` while the other is a Unix timestamp in seconds. There's a comment in `tapdb/sql/migrations/000008_universe_events_timestamp.up.sql` that explains why we have the duplicate field (mainly to avoid this being a breaking DB change).

Reply:
Shouldn't hold back this PR 👍